### PR TITLE
tweaking gulpfile per eslint warnings

### PR DESCRIPTION
**Pull request by pdehaan**
Changed some quotes and spacing for consistency.

Current ESLint output is:

``` sh
$ eslint gulpfile.js

gulpfile.js
   52:19  error  Use path.join() or path.resolve() instead of + to create paths  no-path-concat
  122:27  error  done is defined but never used                                  no-unused-vars
  131:4   error  Don\'t use process.exit(); throw an error instead                no-process-exit

✖ 3 problems (3 errors, 0 warnings)
```

### CODE REVIEW

**Comment by Natim** [comment]
@pdehaan don't you want to ask Travis to run eslint as we do for flake8 on python projects?

File: gulpfile.js [hunk 1]
```diff
@@ -23,12 +23,13 @@ var opt = {
     open: true
   },
   envifyVars: {
-    NODE_ENV:                   process.env.NODE_ENV || DEFAULT_ENV,
-    MAX_ITEMS_PER_PAGE:         process.env.MAX_ITEMS_PER_PAGE,
-    READINGLIST_SERVER_BASEURL: process.env.READINGLIST_SERVER_BASEURL,
```
Comment (n1k0) [inline gulpfile.js:28]: What if I want to keep these aligned for readability?

File: gulpfile.js [hunk 2]
```diff
@@ -170,7 +171,7 @@ gulp.task("watchify", function() {
  * Launch a server with livereload
  */
 gulp.task("watch", ["assets", "js:vendors", "watchify"], function() {
-  gulp.watch(opt.cssAssets,  ["assets:css"]);
```
Comment (n1k0) [inline gulpfile.js:173]: Same remark as above.

**Comment by n1k0** [comment]
The project uses jsxhint, which has its own set of coding rules defined (in .jsxhintrc); while some of these eslint reports are valuable, and I'll definitely merge most of them, I'm not planning to switch to eslint until they natively support JSX. 

Though I'll cherry pick appropriate changes and land them.

**Comment by n1k0** [comment]
(Thanks!)

**Comment by n1k0** [comment]
Cherry-picked changes landed in 9970cd74018cbfad1074b0dc23174638322ed5bb.

**Pull request closed without merging**